### DISCUSSION TITLE
execute_on second time step

### DISCUSSION THREAD

**Discussion by mangerij** [idaholab/moose#23186] [Q&A General] [answered]
I am wondering if it is possible to evaluate an AuxVariable at the second time step (as opposed to the first with `execute_on = 'initial'`).

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can. You ll want to execute on timestep_begin and use a Control (this one I think: https://mooseframework.inl.gov/source/controls/ConditionalFunctionEnableControl.html)
to turn the auxkernel on then off

Guillaume
